[IMP] l10n: Chile new features

samueljlieber opened the pull request:
Task: [#3296044](https://www.odoo.com/web#id=3296044&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

This PR improves the 16.0 Chilean Localization documentation.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/4317).

samueljlieber reviewed (commented):
Hi @vbe-odoo 👋 
I opened this PR as a draft for your review as well as to work out some edits 🙂

I had to update all of the image filenames from what was written in the Google Doc to match our documentation guidelines, please refer to [this document to see the filename changes](https://drive.google.com/file/d/10SAT-JT2P-8KIUi9sVpkt_1t40kvZU2s/view?usp=share_link) also I had to update a few of the _alt_ tags on images because there were duplicates.

I resized and compressed all of the images but there are a few that could be retaken to be more legible:

- [locate-propuesta-f28-report.png](https://github.com/odoo/documentation/pull/4317/files#diff-101cacd83a0b2ee36baa38e9eceb1695c7812ffd24ff6cb133d99a9fe7ac16b3) (chile_edi_49_SP.png)
- [credit-note-document-type.png](https://github.com/odoo/documentation/pull/4317/files#diff-7742c41693ffc252351695387be8b98e321a4e45ddbc5d7bc1b8cf2fb77e2456) (chile_edi_29_SP.png)
- [chilean-cafs.png](https://github.com/odoo/documentation/pull/4317/files#diff-0061783a091e37e8241f28400d049a4701da785815f14bf4ec2785ebad61f1f6) (chile_edi_15_SP.png)

Please let me know your thoughts and if you have any other changes 🙂 Thank you!

samueljlieber commented:
Fixed build error in https://github.com/odoo/documentation/commit/1ef55d12e95e91f7f20c885be0c5eef15d0aa1f9.

samueljlieber commented:
In https://github.com/odoo/documentation/commit/2bc449e9fc7524572f74f6a7513a1ab186433fd4 I [updated images](https://github.com/odoo/documentation/pull/4317#pullrequestreview-1409245264) provided by @masi-odoo and performed a full technical review of the PR. Opening this PR up for final review @StraubCreative 🙂

toaa-odoo commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -264,608 +273,656 @@ Usage
 used for specific partners. After the partners and journals are created and configured, the
-Electronic Receipts are created in the standard way as Electronic Invoice, but the type of document
-:guilabel:`(39) Electronic Receipt` should be selected, like so:
+electronic receipts are created in the standard way as electronic invoice, but the type of document
+:guilabel:`(39) Electronic Receipt` should be selected:
```
```suggestion
electronic receipts are created in the standard way as electronic invoice, but the type of document
:guilabel:`(39) Electronic Receipt` should be selected:
```
Should be explained where to find this

masi-odoo replied:
```suggestion
electronic receipts are created in the standard way as electronic invoice, but the type of document
:guilabel:`(39) Electronic Receipt` should be selected in the invoice form:
```

toaa-odoo commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -875,26 +932,95 @@ Balance Tributario de 8 Columnas
+   :alt: Location of the Propuesta F29 (CL) Report.
 
-You can find this report in :menuselection:`Accounting --> Accounting --> Reports`
+It is possible to set the :guilabel:`PPM` (provisional monthly payments rate) and the
```
```suggestion
It is possible to set the :abbr:`PPM (Provisional Monthly Payments rate) and the
```

toaa-odoo commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -875,26 +932,95 @@ Balance Tributario de 8 Columnas
+
+To activate demo mode in runbot you must:
+
+- Access `runbot.odoo.com <https://runbot.odoo.com/>`_
+- Install the Chilean Localization modules
```
```suggestion
- Access `runbot.odoo.com <https://runbot.odoo.com/>`_.
- Install the Chilean Localization modules.
```

toaa-odoo reviewed (changes requested):
Hi @samueljlieber, this is a long PR, thanks for you work! I made a few comments and suggestions (which are up to you), however there is one recurrent point:

- When explaining that the user should enable one feature or change a field, I think it would be good to also explain *where* the user can find this feature. Considering it is a localization, we can assume this would be a first set-up, and possibly new users without the knowledge of where to find what.

Don't hesitate if you have any questions, and again, thanks a lot for this (very) long PR! :)

samueljlieber commented:
Hi @toaa-odoo, thank you so much for the in-depth review! In https://github.com/odoo/documentation/commit/df03044bf83f9215ff2823fd6bfe847052fd3398 I made the changes you commented/suggested for each of the resolved conversations. The following are still open because I need @masi-odoo help with the content:
- [Add instruction to where the user can setup their company settings](https://github.com/odoo/documentation/pull/4317#discussion_r1227626324)
- [Is this an intentional English screenshot, or should it be Spanish?](https://github.com/odoo/documentation/pull/4317#discussion_r1227644380)
- [Add instruction to where the user can complete the partner form](https://github.com/odoo/documentation/pull/4317#discussion_r1227644777)
- [Add instruction to where the user can configure sales journals](https://github.com/odoo/documentation/pull/4317#discussion_r1227650341)
- [Add instruction to where the user can generate a credit note](https://github.com/odoo/documentation/pull/4317#discussion_r1227675524)
- [Add instruction to where the user can select the Electronic Receipt](https://github.com/odoo/documentation/pull/4317#discussion_r1227738879)

I will push up another commit with these last few items, and then I will request your review again. Thank you! 🙂

masi-odoo commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -5,109 +5,124 @@ Chile
 
-Company Settings
-----------------
+Once the modules are installed, the first step is to set up the company data.
```
```suggestion
Once the modules are installed, the first step is to set up the company data. This can be done in General Settings of Odoo, in the Companies section.

masi-odoo commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -163,97 +169,100 @@ Partner
+customer is important you make sure this value is set:
 
-.. image:: chile/Chile07.png
+.. image:: chile/taxpayer-types.png
```
No reason, there must be an error, but the image should be the next one: https://drive.google.com/file/d/1Jui-hZB1rl-YilX6vtjtgnO4267dObCR/view?usp=sharing

masi-odoo commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -163,97 +169,100 @@ Partner
-based on the Taxpayer Type. This field should be defined in the partner form, when creating
-a customer is important you make sure this value is set:
+In Chile, the document type associated with customers and vendors transactions is defined based on
+the :guilabel:`Taxpayer Type`. This field should be defined in the partner form, when creating a
+customer is important you make sure this value is set:
```
```suggestion
the :guilabel:`Taxpayer Type`. This field should be defined in the partner form, when creating a
customer in the Contacts application, is important you make sure this value is set:
```
Since contact creation is also reviewed in other sections of the Odoo standard documentation I think it should not be a detailed explanation, but added the part referring to the Contacts application

masi-odoo commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -264,608 +273,656 @@ Usage
 When a cancellation or correction is needed over a validated invoice, a credit note must be
-generated. It is important to consider that a CAF file is required for the Credit Note,
-which is identified as document 64 in the SII.
+generated. It is important to consider that a CAF file is required for the credit note, which is
+identified as :guilabel:`Document Type` :guilabel:`61` in the :abbr:`SII (Servicio de Impuestos
+Internos)`.
```
```suggestion
generated. It is important to consider that a CAF file is required for the credit note, which is
identified as :guilabel:`Document Type` :guilabel:`61` in the :abbr:`SII (Servicio de Impuestos
Internos)`. Please refer to :doc:`Chile/Configuration/CAF` for more information.
```

Referencing section with this information

masi-odoo commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -264,608 +273,656 @@ Usage
 
-**Use Documents**: This field is used to define if the journal will use Document Types. It is only
-applicable to Purchase and Sales journals that can be related to the different sets of document
-types available in Chile. By default, all the sales journals created will use documents.
-
-.. image:: chile/Chile41.png
-   :align: center
-   :alt: Document type configuration on Journal.
+:guilabel:`Use Documents`: This field is used to define if the journal will use document types. It
+is only applicable to purchase and sales journals that can be related to the different sets of
+document types available in Chile. By default, all the sales journals created will use documents.
```
I cannot suggest directly from the lines :( will comment instead.

Configuration
-------------

To create a Sales Journal please refer to Accounting ‣ Configuration ‣ Journals.

When creating sales journals the next information must be filled in:

samueljlieber commented:
Hi @toaa-odoo this PR is ready for another look, @masi-odoo (thank you! 🙂) and I addressed the [remaining suggestions](https://github.com/odoo/documentation/pull/4317#issuecomment-1599030404) you had in commit https://github.com/odoo/documentation/commit/65abb18d2f9dc227b76aa33b03daabf01d954628.

Thank you all for the continued effort on this PR 🙏

samueljlieber commented:
Resolved merge conflict in https://github.com/odoo/documentation/commit/b3668466f70da7f629856acea26b9156b30c0535 due to a warning admonition added to `chile.rst` in https://github.com/odoo/documentation/pull/5026.

samueljlieber commented:
Hi @jcs-odoo thank you for point that out– I compressed the 3 that I missed previously and double checked all the others in https://github.com/odoo/documentation/commit/768220b8ee1bdfa8b4eda7d4a186bfc94698fdfc. This PR should be all set for you @jcs-odoo 🙏 thank you in advance!

jcs-odoo commented:
@xpl-odoo can you review this, please? I just changed slightly the structure... You can check my changes and see if they make sense, otherwise, feel free to revert them.

Thank you a lot!

xpl-odoo reviewed (changes requested):
Hi @samueljlieber !
I started an in-depth review, but I'm wondering if it might be too detailed for the purpose of this PR. I stopped around line 100, as it seems you didn't go for a full rewrite. Let me know if it's what you were looking for.

samueljlieber commented:
In https://github.com/odoo/documentation/commit/e3b37393064ffe011e24321c314f2afe8595ca27 and https://github.com/odoo/documentation/commit/b01d9394c34eec32656f67aad329b9d9326f26ed I implemented @xpl-odoo's suggestions, updated images and reworked the _Accounting settings_ section. Tagging @xpl-odoo for a full review, thank you in advance!

samueljlieber commented:
I am making additional edits to improve the depth of content even further and add more textual instructions to replace many images.

samueljlieber commented:
Made additional edits in https://github.com/odoo/documentation/commit/0aa16baf37ccfddc601db310fd4e052731a249a8 and https://github.com/odoo/documentation/commit/27de96d6aed77b9d657a1ea270fcaf7ee57859d6 to improve context by adding addition written instructions and reduced the number of images.

samueljlieber commented:
Hi @odoo/be-doc-review this PR is ready for your review 🙂

xpl-odoo commented:
Hi @samueljlieber!
I'll push some changes in a second commit to proceed faster, directly adding comments in the .rst. if necessary.
I'll then let you check and r+.

samueljlieber commented:
Thank you @xpl-odoo for [your comments](https://github.com/odoo/documentation/commit/aa781e62399279010715dd8c1d413da7943ea43e)! I worked with @masi-odoo to implement them in https://github.com/odoo/documentation/commit/631d8c1d339cf86f4b3ba7fce127cb23220fd394. 

I will now tag @odoo/us-doc-review for a final look 🙏
c.c. @StraubCreative

StraubCreative commented:
cc: @erla-odoo CL l10n documentation 😄

StraubCreative commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,951 @@
-Electronic Invoice Data
-***********************
+The :abbr:`DTE (Documentos Tributarios Electrónicos)` :guilabel:`Email Box Electronic Invoicing` can
+be defined to receive your customers' claim and acceptance emails. Enabeling this option is
```
Typo
```suggestion
be defined to receive your customers' claim and acceptance emails. Enabling this option is
```

StraubCreative commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,951 @@
+The :abbr:`DTE (Documentos Tributarios Electrónicos)` :guilabel:`Email Box Electronic Invoicing` can
+be defined to receive your customers' claim and acceptance emails. Enabeling this option is
+necessary if you want to use *Email Box Electronic Invoicing* as the :abbr:`DTE (Documentos
+Tributarios Electrónicos)` incoming email server. To do so, click :guilabel:`Configure DTE Incoming
+Email`. Then, click :guilabel:`New` to add a server and fill in the following fields:
```
Just spoke to a CL l10n expert here in SF office. We need this **important** admonition here as we receive a lot of support tickets in relation to this cc: @erla-odoo 
```suggestion
Tributarios Electrónicos)` incoming email server.

.. important::
   In order to receive your SII documents, it's necessary to set up your own email server.
   More information on how to do this can be found in this documentation:
   :doc:`../../general/email_communication/email_servers`

Begin by, clicking :guilabel:`Configure DTE Incoming Email`. Then, click :guilabel:`New` to add a
server and fill in the following fields:
```

samueljlieber replied:
@StraubCreative the instructions in this section are for setting up an _Incoming Email Server_, I am not sure if linking to the `email_servers.rst` doc is correct since the instructions in the **DTE incoming email server** section are specific to l10n_CL. 

I agree that an **Important** admonition is required to highlight this step, but the instructions are already within this section, no?

cc: @masi-odoo @elre-odoo

erla-odoo replied:
You could leave it without the link, I suggested having it because some people do want more information regarding setting up the email servers so having the link would lower the amount of tickets submitted for that.

samueljlieber replied:
@erla-odoo I will leave the link then! I wanted to make sure the link was relevant! 🙂 Thanks!

StraubCreative commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,951 @@
 
-Fiscal Positions
-~~~~~~~~~~~~~~~~
+- Name
```
```suggestion
- :guilabel:`Name`
```

StraubCreative commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,951 @@
 
+Configuring partner contacts is also required to send :abbr:`SII (Servicio de Impuestos Internos)`
+electronic invoices. Open the :menuselection:`Contacts` app to do so and fill in the following
+fields:
```
be specific 😉 
```suggestion
fields on a new or existing contact record's form.
```

StraubCreative reviewed (changes requested):
Hi @samueljlieber @masi-odoo 

This is a really great improvement to what we had before. Well done 👏 

For the sake of time I need to pause my review here for the moment. Below you'll find a  few things I caught which I'll be pushing up the changes for in another commit. They are as follows:

- specificity around headings and some body content
- center-aligned all images
- h3 --> h2 formatting for "configuration" headings since it makes them visually easier to read and find and more closely follows the [Document Structure guidelines](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#document-structure). I left **Usage and testing** and **Financial reporting** sections collapsed, as they were since they are advanced reading that can be clicked open as needed by the reader.
- admonition block added by one of our l10n experts regarding email servers

I think we could use another pass around specificity of language and navigational information. 
- There are times where there are good UI instructions, but it's unclear _where we actually are_ in Odoo
- There are times where we're skipping a step or not actually saying what things are. 
Example:
```Open the :menuselection:`Contacts` app to do so and fill in the following
fields:```
You can't actually do this instruction above without first specifying to open a Contacts _record_ and then editing the record's the _form_. 😅 

I'm going to pull in some other writers on my team to review the remaining contents and help close this out.

Again, great work, we're almost done I think 👍

StraubCreative commented:
[39d00d8](https://github.com/odoo/documentation/commit/39d00d87d063b49d72a663418ebcd14b6b81f806): addressed CRs from [last review](https://github.com/odoo/documentation/pull/4317#pullrequestreview-1698334478) 

Feel free to **Resolve conversation** on individual change requests as you read them (since they should be fixed now)

samueljlieber commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,951 @@
 
-Electronic Invoice Data
-***********************
+The :abbr:`DTE (Documentos Tributarios Electrónicos)` :guilabel:`Email Box Electronic Invoicing` can
```
Yes we are still in the **Accounting ‣ Configuration ‣ Settings ‣ Chilean Localization** here, 
<img width="640" alt="Screenshot 2023-10-26 at 10 32 14 AM" src="https://github.com/odoo/documentation/assets/36018073/e5fe2db5-6b61-4745-a7d7-8bf7458387e9">

@samueljlieber restate where the user is in the **DTE incoming email server** section

larm-odoo reviewed (commented):
Hello- I have some suggestions just to clarify the information a bit.

StraubCreative commented:
Thanks for the help @larm-odoo !
@samueljlieber can you implement please and then tag @Felicious to do a review?

samueljlieber commented:
Thank you @larm-odoo! Hi @Felicious can you please review, Lara got to about line 300, so feel free to start there 🙏
Thank you!

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
+   - :guilabel:`Accepted with objections`: Indicates the invoice information is correct, but a minor
+     issue was identified, nevertheless the document is now fiscally valid and it's automatically
+     sent to the customer.
+   - :guilabel:`Rejected`: Indicates the information in the invoice is incorrect and needs to be
+     corrected, the detail of the issue is received in the emails you registered in the :abbr:`SII
+     (Servicio de Impuestos Internos)`, if it is properly configured in Odoo, the details are also
+     retrieved in the chatter once the email server is processed.
```
```suggestion
   - :guilabel:`Rejected`: Indicates the invoice information is incorrect and must be corrected.
     Details are sent to emails you registered in the :abbr:`SII (Servicio de Impuestos Internos)`.
     If it is properly configured in Odoo, the details are also retrieved in the chatter once the
     email server is processed.
```

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
+
+     If the invoice is rejected please follow these steps:
+
+      - Change the document to :guilabel:`Draft`.
+      - Make the required corrections based on the message received from the :abbr:`SII (Servicio de
+        Impuestos Internos)` in the chatter.
+      - Post the invoice again.
```
```suggestion
      #. Change the document to :guilabel:`Draft`.
      #. Make the required corrections based on the message received from the :abbr:`SII (Servicio
         de Impuestos Internos)` in the chatter.
      #. Post the invoice again.

```

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
-4. The field DTE acceptation status is updated automatically.
+#. :guilabel:`DTE Partner Status` changes to :guilabel:`Sent`.
+#. The customer must send a reception confirmation email.
+#. Subsequently, if all the commercial terms and invoice data are correct, they will send the
+   acceptance confirmation, otherwise they send a claim.
```
Suggestions for brevity. Also, not really sure who "they" are in the original sentence. Might be useful to specify

```suggestion
#. Subsequently, if commercial terms and invoice data are correct, an acceptance confirmation is
   sent; otherwise, a claim is sent.
```

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
+There are multiple reasons behind a rejection from the :abbr:`SII (Servicio de Impuestos Internos)`,
+but these are some of the common errors you might have and how to solve them:
+
+- | **Error:** `RECHAZO- DTE Sin Comuna Origen`
+  | **Hint:** Make sure the company address is properly filled including the state and city.
```
out of curiosity, what's this formatting with the | here for, Sam? (:

samueljlieber replied:
@Felicious  the `|` character [forces a line break](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks). So it can be helpful to use in a list, if you want to break a line in a bullet point :)

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
 
-In case you need to cancel or invalid an Invoice, use the button Add Credit note and select Full
-Refund, in this case the SII reference Code is automatically set to: Anula Documento de referencia.
+In case you need to cancel or invalid an invoice, use the button :guilabel:`Add Credit note` and
+select :guilabel:`Full Refund`, in this case the :abbr:`SII (Servicio de Impuestos Internos)`
+reference code is automatically set to :guilabel:`Anula Documento de referencia`.
```
Is this where I'd find an invoice? I don't see the "Full Refund" option but I might be missing some configuration

```suggestion
In case you need to cancel or invalid an invoice, navigate to :menuselection:`Accounting -->
Customers --> Invoices` and select the desired invoice. Then, use the button :guilabel:`Add Credit
Note` and select :guilabel:`Full Refund`, in this case the :abbr:`SII (Servicio de Impuestos
Internos)` reference code is automatically set to :guilabel:`Anula Documento de referencia`.
```
![image](https://github.com/odoo/documentation/assets/23611711/3cd1a985-9bc9-41e3-b225-928828dca962)

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
 .. important::
-   It's important to define the default credit account in the Sales journal as it
-   is taken for this use case in specific.
+   It's important to define the :guilabel:`Default Credit Account` in the sales journal as it is
+   taken for this use case in specific.
```
```suggestion
   Make sure to define the :guilabel:`Default Credit Account` in the sales journal specifically for
   this use case.
```

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
-As part of the Chilean localization, besides creating credit notes from an existing document you
-can also create debit Notes. For this just use the button “Add Debit Note”. The two main use cases
-for debit notes are detailed below.
+As part of the Chilean localization, besides creating credit notes from an existing document you can
+also create debit notes. For this just use the button :guilabel:`Add Debit Note`. The two main use
+cases for debit notes are detailed below.
```
optional wording suggestion for brevity
```suggestion
In Chilean localization, debit notes, in addition to credit notes, can be created using the
:guilabel:`Add Debit Note` button, with two main use cases.
```

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
 
-The most common use case for debit notes is to increase the value of an existing invoice, you
-need to select option 3 in the field Reference code SII:
+The most common use case for debit notes is to increase the value of an existing invoice, you need
+to select option :guilabel:`3. Corrige el monto del Documento de Referencia` for the
+:guilabel:`Reference Code SII` field.
```
```suggestion
The primary use case for debit notes is to increase the value of an existing invoice. To do so,
select option :guilabel:`3. Corrige el monto del Documento de Referencia` for the
:guilabel:`Reference Code SII` field.
```

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
-In Chile the debits notes are used to cancel a validated Credit Note, in this case just
-select the button Add debit note and select the first option in the wizard: *1: Anula
-Documentos de referencia.*
+In Chile, the debits notes are used to cancel a valid credit note. In this case, just select the
+:guilabel:`Add Debit Note` button and then select the :guilabel:`1: Anula Documentos de referencia`
+option for the :guilabel:`Reference Code SII` field.
```
optional wording suggestion
```suggestion
In Chile, debits notes are used to cancel a valid credit note. TO do this, click the :guilabel:`Add
Debit Note` button and select the :guilabel:`1: Anula Documentos de referencia` option for the
:guilabel:`Reference Code SII` field.
```

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
 
-As part of the Chilean localization, you can configure your Incoming email server as the same you
-have register in the SII in order to:
+As part of the Chilean localization, you can configure your incoming email server as the same you
+have registered in the :abbr:`SII (Servicio de Impuestos Internos)` in order to:
```
Personally, I thought the wording "as the same" sounded off, but I can't articulate why
```suggestion
As part of the Chilean localization, you can configure your incoming email server to match the one
you have registered in the :abbr:`SII (Servicio de Impuestos Internos)` in order to:
```

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
 In case there is a commercial issue or the information is not correct on your vendor bill, you can
-Claim the document before validating it, using the button: Claim, once this is done the DTE
-Acceptation Status change to: Claim and an email of acceptance is sent to the vendor.
+claim the document before validating it, using the :guilabel:`Claim` button, once this is done, the
```
The original sentence was a little long so I'm splitting it in two here (:
```suggestion
claim the document before validating it, using the :guilabel:`Claim` button. Once this is done, the
```

Felicious commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
-Claim the document before validating it, using the button: Claim, once this is done the DTE
-Acceptation Status change to: Claim and an email of acceptance is sent to the vendor.
+claim the document before validating it, using the :guilabel:`Claim` button, once this is done, the
+:guilabel:`DTE Acceptation Status` changes to :guilabel:`Claim` and an email of rejection is sent to
+the vendor.
```
```suggestion
:guilabel:`DTE Acceptation Status` changes to :guilabel:`Claim` and a rejection email is sent to the
vendor.
```

Felicious reviewed (approved):
Hi @samueljlieber! Just finished adding comments to lines 225 - 744. Once I saw I had to install the Delivery guide module, I figured it was time to stop for today. 

Great job on this thicc-y doc-- I only had minor wording suggestions and a few places where I wasn't sure where we were, that I pointed out in my comments. I can probably tackle another chunk on another day, but for now, my brain is checked out. It's amazing how you made all of these changes and added all of these new screenshots. The doc is in a great place! The sections are clear and the writing flows well between the sections. I can tell you put in a lot of effort :blush:

samueljlieber commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
-that you only need one active CAF per document type and it will be applied on all journals.
+Once the :abbr:`CAF (Folio Authorization Code)` files have been acquired from the :abbr:`SII
+(Servicio de Impuestos Internos)` portal, they need to be uploaded in the database by navigating to
+:menuselection:`Accounting --> Chilean SII --> CAFs`. Then, click the :guilabel:`New` button to
```
I like the way you did it here- formatting subheadings in a menu with a colon. I think both ways are correct!

samueljlieber commented:
Thank you for the review @Felicious! I implemented most of your suggestions, all were very helpful 🙂
@StraubCreative I think this PR is ready to go 🚀

StraubCreative commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
 
-Electronic Receipt
+Delivery guide will have fiscal elements that indicate that the document is fiscally valid when
+printed (if hosted in Odoo SH or on Premise remember to manually add the :guilabel:`pdf417gen`
```
Italicize Odoo products, spelling
```suggestion
printed (if hosted in _Odoo SH_ or _On-premise_ remember to manually add the :guilabel:`pdf417gen`
```

StraubCreative commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
-Sales Order, proceed to validate the receipt. By default, Electronic Invoice is selected as the
-Document Type, however in order to validate the receipt correctly, make sure to edit the Document
-Type and change to Electronic Receipt.
+When all of the electronic receipt information is filled, either manually or automatically from a
+sales order, proceed to validate the receipt. By default, :guilabel:`Electronic Invoice` is selected
```
grammar
```suggestion
When all of the electronic receipt information is filled, manually (or automatically) proceed to validate the receipt from the sales order. By default, :guilabel:`Electronic Invoice` is selected
```

StraubCreative commented on content/applications/finance/fiscal_localizations/chile.rst:
```diff
@@ -2,898 +2,998 @@
+information required by it.
 
-A list of daily reports is displayed with all daily DTE sent to SII.
+Contact configurations
```
There's no written content here?

StraubCreative reviewed (approved):
Hi @samueljlieber @masi-odoo @vbe-odoo 

Approving with comments so we can get this in the hands of Chilean users. We can deploy the updates as they are, however I have a number of items to look at, listed below, perhaps more fitting for a separate PR. The most consistent items I saw revolved around grammar and lean content (the usual suspects 😅)

Your call @samueljlieber 🤙 

@robodoo delegate=samueljlieber

samueljlieber commented:
Thank you for the thorough review @StraubCreative, I implemented all of the resolved suggestions in https://github.com/odoo/documentation/commit/f83484b13f2efc703538bca16082ac5ed9103d56 and left the ones that I will need to work on in another PR unresolved.

c.c. @masi-odoo 

@robodoo r+
